src/lib/elements/forms/inputSelectSearch.svelte

```diff
@@ -20,9 +20,10 @@
     export let placeholder = '';
     export let required = false;
     export let disabled = false;
+    export let fullWidth = false;
     export let autofocus = false;
     export let interactiveOutput = false;
-    // Input value
+    export let stretch = true;
```

Comment:
`fullWidth` gives it the `inline-size: 100%` while `stretch` is `flex:1`

Reply:
@ArmanNik it's still a little confusing because, as someone who's just using the InputSelectSearch component, I'm not sure when/why I would use one or the other.